### PR TITLE
Improve the readme usage examples

### PR DESCRIPTION
**Pull request by justsml**
Thanks Sindre for this amazing library!!!

* Used comments to label examples w/ headers in markdown.
* Added a promise example to help a more diverse set of users.

I live-coded this in the spirit of Hacktoberfest! 

My goal was to become youtube famous https://youtu.be/z7svdNlAJpY ... 

And help with your readme 😇

### CODE REVIEW

**Comment by sindresorhus** [comment]
Thanks for helping us improve the readme 🙌

**Comment by sindresorhus** [comment]
> Added a promise example to help a more diverse set of users.

I've removed this. We're not gonna document JavaScript features. It's expected for users to know/learn async/await.

**Comment by justsml** [comment]
That's fair enough. 👍  

Really appreciate all your work for the community ❤️

File: readme.md [hunk 1]
```diff
@@ -41,16 +41,21 @@ const execa = require('execa');
...
+execa('echo', ['unicorns']).stdout.pipe(process.stdout);
+```
 
+### Handling Errors
 
+```js
+const execa = require('execa');
+
+(async () => {
+	// Pipe the child process stdout to the current stdout
```
Comment (ehmicky) [inline readme.md]: This comment should be removed.

Comment (justsml) [reply readme.md]: Fixed! Good spotting @ehmicky !

**Comment by ehmicky** [comment]
Thanks @justsml, this is merged! :tada:

**Pull request merged**